cdap-common/src/main/java/co/cask/cdap/common/logging/LoggingContextAccessor.java

```diff
@@ -51,6 +51,13 @@ public static void setLoggingContext(LoggingContext context) {
   }
 
   /**
+   * Removes the logging context
+   */
+  public static void resetLoggingContext() {
+    loggingContext.remove();
```

Comment:
In the `setLoggingContext` in this class, it does

```java
MDC.setContextMap(context.getSystemTagsAsString());
```

Isn't that need to be reset?